### ISSUE TITLE
As a Developer, I'd like to be able to get RepaymentRate() data so that I can integrate it into my app, for example to create a chart that shows trends over time

### ISSUE THREAD

**Issue by LisaGee** [18F/college-choice#183]
I'd like to make a function call:
RepaymentRate()

Inputs:
* School: All or IndividualSchool (any identifier referenced in #221)
* Year: 1996-current
* Student Characteristic: All, Completers, NonCompleters, LowIncome, MedIncome, HighIncome, Dependent, Independent, Pell, NoPell, Female, Male, FirstGen, NotFirstGen
* Timeframe: 1yr, 3yr, 5yr, 7yr

Outputs:
* RepaymentRate
* NumInCohort

**Comment by ultrasaurus** [comment]
If we want to introduce the concept of the "current" year, we should probably break that out as a separate story.  I think that would be great, but probably lower priority than other things.

**Comment by ultrasaurus** [comment]
assigning to Lisa to address above comment and waiting on info from #184

**Comment by LisaGee** [comment]
Repayment Rate data found in rows: 1381-1436 in Data Dictionary.
# of students in each cohort found in rows: 1589-1644.

**Comment by LisaGee** [comment]
Diego wrote this as a sample of what this call might look like:

  Owner
diego- commented just now
implement the function call for ticket #183:

http://ccapi-prod.18f.gov/school?ope_id=00143400&year=2012&fields=3_yr_repayment_suppressed.income.low_income,3_yr_repayment.overall

**Comment by ultrasaurus** [comment]
This is the same kind of thing as 184 :https://ccapi-prod.18f.gov/school?id=155937&fields=id,name,RPY_3YR_N,RPY_3YR_RT_SUPP,year
